.ci/install_clear_image.sh

```diff
@@ -0,0 +1,50 @@
+
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 CLEAR_RELEASE PATH"
```

Comment:
How will the user know what value to provide for `CLEAR_RELEASE`?

It would be helpful if the script supported `latest` (either to simply show the newest-available release version and exit, or actually use that version). Same comment applies to `install_clear_kernel.sh`).